README.md

```diff
@@ -30,6 +30,39 @@ To build an individual package in response to changes within the `src` directory
 ```sh
 yarn dev
 ```
+
+### Testing
+Tests are run with [Mocha](https://mochajs.org/), [Chai](https://www.chaijs.com/), and [Sinon](https://sinonjs.org/) with the [Karma test runner](https://karma-runner.github.io/latest/index.html) in a browser environment. The following browsers are currently tested:
+
+- Chrome
+
+To run tests for a specific web component package, navigate to its directory and use one of the following commands based on the desired testing flow:
+
+- To run the tests for a specific package once:
+```sh
+yarn test
+```
+- To run the tests for a specific package and watch the `src` and `test` directories to rerun the tests after any changes:
+```sh
+yarn tdd
+```
+
+Test coverage is also provided using [Istanbul](https://github.com/istanbuljs/istanbuljs).
+
+When starting a new component, create new `setup.ts` and `utils.ts` files in its new `tests` directory. These files will make the same files from the root `test-tools` directory available through imports and exports.
+
+Ex.
+- utils.ts
+```js
+export * from '../../../../../test-tools/utils';
+```
+- setup.ts
+```js
+import '../../../../../test-tools/setup';
```

Comment:
Having to traverse 5 levels up is a red flag for me. We need to get some of these high-level folders (eg. `test-tools/`) on the path so we can import them directly.

This need not happen as part of this story/PR, but I wanted to point it out.

Reply:
As discussed, this will be reinvestigated later.